src/sass/components/_header.scss

```diff
@@ -144,79 +121,98 @@ header {
             border-top: 4px solid #ffffff;
             font-weight: 500;
             letter-spacing: 1px;
-
             &:hover,
             &.active {
               background-color: $primary;
               color: #ffffff;
               border-color: $primary;
             }
-
             &.active {
               border-color: $text-color;
             }
           }
         }
       }
     }
-
     .search-form {
       border: 1px solid $form-border-color;
       display: inline-flex;
       height: 45px;
       color: $text-color;
-
       .category {
         border-right: 1px solid $form-border-color;
         display: flex;
         align-items: center;
         position: relative;
-
         i:first-child {
           color: $primary;
           position: absolute;
           left: 10px;
           z-index: 0;
         }
-
         i:last-child {
           position: absolute;
           right: 10px;
           z-index: 0;
         }
-
-        select {
-          -webkit-appearance: none;
-          border: 0;
-          background-color: transparent;
-          outline: none;
-          padding: 5px 30px 5px 35px;
-          font-size: 14px;
-          position: relative;
+        .category-menu {
+          width: 200px;
+          text-align: center;
+          position: absolute;
+          top: 10px;
+          list-style: none;
+          padding: 0;
           z-index: 1;
-        }
-      }
-
-      .search-field {
-        display: flex;
-        align-items: center;
-        position: relative;
-
-        input {
-          border: 0;
-          padding-left: 10px;
-          font-size: 14px;
-          outline: none;
-
-          &::placeholder {
-            color: $text-color;
+          li {
+            display: none;
+            position: relative;
+            background-color: #fff;
+            line-height: 43px;
+            cursor: pointer;
+            &:hover {
+              display: block;
+            }
+            a {
+              text-decoration: none;
+              color: #2a2a2a;
+              &:hover {
+                background-color: $primary;
+                color: #fff;
+              }
+            }
+            ul {
+              display: none;
+              position: absolute;
+              left: 200px;
+              top: 0;
+              width: 200px;
+              padding: 0;
+              li {
+                &:hover a {
+                  color: #ffffff;
```

Comment:
Można użyć skróconej składni `#fff` dodatkowo fajnie byłoby to wyciągnąć do zmiennej